internal/apis/certmanager/v1alpha3/zz_generated.conversion.go

```diff
@@ -863,8 +863,8 @@ func autoConvert_certmanager_CertificateSpec_To_v1alpha3_CertificateSpec(in *cer
 	out.RenewBefore = (*v1.Duration)(unsafe.Pointer(in.RenewBefore))
 	out.DNSNames = *(*[]string)(unsafe.Pointer(&in.DNSNames))
 	out.IPAddresses = *(*[]string)(unsafe.Pointer(&in.IPAddresses))
-	out.URISANs = *(*[]string)(unsafe.Pointer(&in.URISANs))
-	out.EmailSANs = *(*[]string)(unsafe.Pointer(&in.EmailSANs))
+	// WARNING: in.URIs requires manual conversion: does not exist in peer-type
+	// WARNING: in.EmailAddresses requires manual conversion: does not exist in peer-type
```

Comment:
what's this?

Reply:
I also added manual conversion functions for these fields.
This is caused by me renaming URISANs to URIs in our internal type.
This makes it easier to reason about the internal type (because its field names match the public API).
For older API versions, this requires adding some conversion functions (which I did add).

Reply:
I'll move these changes to another PR.